cmd/bucket-replication.go

```diff
@@ -300,12 +300,12 @@ var (
 	// but eventually can be tuned in future,
 	// take only half the CPUs for replication
 	// conservatively.
-	globalReplicationConcurrent = runtime.GOMAXPROCS(0) / 2
```

Comment:
We can't use all CPUs @poornas for replication we have to only use 50% - when CPU is 1 it should be just 1